docs/cli/caliban_shell.rst

```diff
@@ -84,3 +84,11 @@ prevent you from getting too far.
    will use your ``$SHELL`` environment variable to pick a default; to override
    the default, you can always pass the ``--shell`` argument, like this:
    ``caliban shell --shell bash``.
+
+One potential issue resulting from the fact that your home directory will mount
```

Comment:
if you do something like

```
If you do

.. WARNING:: For some reason I don't understand, if you pass ``-v`` instead of
   ``--volume``\ , as in ``--docker_run_args "-v mydir:containerdir"``\ , the
   argument parser in Caliban will break. Use ``--volume`` and you'll be set!
```

you can get a nice yellow box;

![image](https://user-images.githubusercontent.com/69635/88326140-f860f380-cce2-11ea-9428-d1cb4e165420.png)